functions/data/medicationCodes.json

```diff
@@ -4,6 +4,7 @@
     "medications": [
       {
         "code": "221124",
+        "brandNames": ["Toprol XL", "Kapspargo Sprinkle"],
```

Comment:
As noted in the discussion with Alex, we can only focus on one brand name for now, at least for the seeded data that we provide.